napari/layers/points/points.py

```diff
@@ -1474,6 +1475,15 @@ def _get_state(self):
                 'shown': self.shown,
             }
         )
+        for old_key in DEPRECATED_PROPERTIES:
+            new_key = old_key.replace('edge', 'border')
+            if new_key in state:
+                state.deprecate_with_replacement(
+                    old_key,
+                    new_key=new_key,
+                    version='0.6.0',
+                    since_version='0.5.0',
+                )
```

Comment:
This is the second place where this deprecation is defined. In https://github.com/andy-sweet/napari/pull/69 I have proposed a more generic solution to avoid multiple place to be edited.